25lessons/script.js

```diff
@@ -0,0 +1,103 @@
+//1
+let number1 = +prompt("Enter first number");
+let number2 = +prompt("Enter second number");
+function number(number1, number2) {
+  if (number1 < number2) {
+    return -1;
+  } else if (number1 > number2) {
+    return 1;
+  } else {
+    return 0;
+  }
+}
+alert(number(number1, number2));
+
+//2
+let factorialNumber = +prompt("Enter number please:");
+function factorialOfNumber(factorialNumber) {
+  let number = 1;
+  for (let i = 1; i <= x; i++) {
+    number *= i;
+  }
+  return number;
+}
```

Comment:
всередині в тебе `x`, тому й аргумен повинен бути `x`
```suggestion
function factorialOfNumber(x) {
  let number = 1;
  for (let i = 1; i <= x; i++) {
    number *= i;
  }
  return number;
}
```